document-warehouse/test/set-acl.test.js

```diff
@@ -19,28 +19,46 @@ const cp = require('child_process');
 const path = require('path');
 const assert = require('assert');
 
+const {PoliciesClient} = require('@google-cloud/iam').v2;
+const {ProjectsClient} = require('@google-cloud/resource-manager').v3;
+const iamClient = new PoliciesClient();
+const projectClient = new ProjectsClient();
+
 const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
 const cwd = path.join(__dirname, '..');
-const PROJECT_ID_PASS = 'YOUR_PROJECT_ID';
+
 const PROJECT_ID_FAILED = 'PROJECT_ID_WITHOUT_ACL';
-const LOCATION = 'us';
 const POLICY_ROLE = 'roles/contentwarehouse.documentAdmin';
 const POLICY_MEMBER = 'user:xxxx@example.com';
-const USER_ID = 'user:xxxx@example.com';
 const DOCUMENT_ID = 'YOUR_DOCUMENT_ID';
 
 describe('Set document acl', () => {
+  let projectNumber;
+  const location = 'us';
+
+  async function getProjectNumber() {
+    const projectId = await iamClient.getProjectId();
+    const request = {name: `projects/${projectId}`};
+    const project = await projectClient.getProject(request);
+    const resources = project[0].name.toString().split('/');
+    projectNumber = resources[resources.length - 1];
+  }
```

Comment:
It might make sense to put all acl tests in the same file so they can all use this function